mne/simulation/metrics/tests/test_metrics.py

```diff
@@ -0,0 +1,249 @@
+# Authors: Kostiantyn Maksymenko <kostiantyn.maksymenko@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+import os.path as op
+import warnings
+
+import numpy as np
+from numpy.testing import assert_almost_equal
```

Comment:
`assert_almost_equal` is deprecated. use `allclose`. see:https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_almost_equal.html